layouts/partials/nav/left-nav.html

```diff
@@ -63,7 +63,7 @@
                                         </a>
                                         {{/* 4th level, hide */}}
                                         {{ if .HasChildren }}
-                                            <ul class="list-unstyled sub-menu {{ if not (in (slice "observability_pipelines_reference_processing_language" "observability_pipelines_vector_configuration") .Identifier) }}d-none{{ end }}">
+                                            <ul class="list-unstyled sub-menu {{ if not (in (slice "custom_instrumentation" "observability_pipelines_reference_processing_language" "observability_pipelines_vector_configuration") .Identifier) }}d-none{{ end }}">
```

Comment:
```suggestion
                                            <ul class="list-unstyled sub-menu {{ if not (in (slice "automatic_instrumentation" "custom_instrumentation" "observability_pipelines_reference_processing_language" "observability_pipelines_vector_configuration") .Identifier) }}d-none{{ end }}">
```
Would this let us show the submenu for Automatic Instrumentation too?